bin/mplx_worklist.py

```diff
@@ -116,8 +116,13 @@ def read_input(input_file):
     logger.debug('master worksheet name: %s', master_worksheet.title)
     row_iter = iter(master_worksheet.rows)
     header_row = next(row_iter)
-    # Read header row and parse into column names.
+    # Read header row and parse into column names
     column_names = [c.value for c in header_row]
+    # remove NoneType column names
+    column_names = remove_none_type_col(column_names)
+    for column_name in column_names:
```

Comment:
Unecessary for loop. The `any` function was looping over a generator expression.